common/util/file_util.cc

```diff
@@ -174,7 +174,18 @@ absl::Status SetContents(absl::string_view filename,
 }
 
 std::string JoinPath(absl::string_view base, absl::string_view name) {
-  return absl::StrCat(base, "/", name);
+  // TODO: this will certainly be different on Windows
```

Comment:
I assume the idea is not to worry about that for now as it will be fixed when switching to c++17?

Reply:
Yes. I'll work on using `std::filesystem` soon, so this is mostly a reminder.